### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by mattmunich**
- Adds .editorconfig to enforce consistent styling
- Fix all formatting & analysis errors

### CODE REVIEW

File: src/Hyde/Hyde.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <Description>Object to Entity mapper for Windows Azure</Description>
-    <Copyright>Copyright TechSmith Corporation 2017</Copyright>
+    <Copyright>Copyright TechSmith Corporation 2017-2022</Copyright>
```
Comment (jefflward) [inline src/Hyde/Hyde.csproj:5]: Is this necessary. I was under the opinion that a single date implied `2017-NOW`

**Review by jefflward — APPROVED** [review]
👍🏻

File: src/Hyde/Hyde.csproj (hunk 1 above)
Comment (mattmunich) [reply src/Hyde/Hyde.csproj]: Good call -- dropped commit 5226351

**Comment by mattmunich** [comment]
[Update Hyde in cloud services for potential embed 5xx issues](https://trello.com/c/wXAWu6kj/862-update-hyde-in-cloud-services-for-potential-embed-5xx-issues)

**Pull request merged**